.gitmodules

```diff
@@ -42,3 +42,9 @@
 [submodule "src/tools/miri"]
 	path = src/tools/miri
 	url = https://github.com/solson/miri.git
+[submodule "src/dlmalloc"]
+	path = src/dlmalloc
+	url = https://github.com/alexcrichton/dlmalloc-rs
+[submodule "src/binaryen"]
+	path = src/binaryen
+	url = https://github.com/WebAssembly/binaryen
```

Comment:
There's a small inconsistency here: All the other crates use `https://github.com/owner/repo.git` instead of `https://github.com/owner/repo`